src/fondant/data_io.py

```diff
@@ -196,6 +206,12 @@ def write_dataframe(self, dataframe: dd.DataFrame) -> None:
 
         dataframe.index = dataframe.index.rename("id").astype("string")
 
+        # logging.info("Visualizing task graph...")
```

Comment:
If I have custom code that I use for testing but don't want to commit I usually use 
```
git add -p <filename>
```
which allows you to only commit chunks of the file, can be also done in the IDE. 

Then if I want to switch to another branch and not lose the testing code, I stash the changes under a descriptive name

```
git stash save datacomp-scale-test
```

You can then fetch it back using 
```
git stash apply <stash_id>
```